analysis/source/plot/create_plots.R

```diff
@@ -23,20 +23,21 @@ main <- function() {
   
 }
 
-plot_hist <- function(data_path, out_path, var_name, in_file, out_file, height = 5, width = 6){
+plot_hist <- function(data_path, out_path, var_name, in_file, out_file, height = 4, width = 5){
```

Comment:
@kobleary does `height = 4` and `width = 5` render well in **LaTeX**?

Reply:
@nateschor yep, we've used this sizing for another paper and it worked great.